libraries/AP_Mount/AP_Mount_Alexmos.cpp

```diff
@@ -23,6 +23,7 @@ void AP_Mount_Alexmos::update()
         return;
     }
 
+    get_angles_ext();
```

Comment:
What does this call do?

Reply:
This sends a query to Basecam (Alexmos) requested the extended angles message.  This message has the current accelerometer and encoder angles.  We make this request here and then move on.  When the response arrives asynchronously we save those values in the _current_angle variable.  (See AP_Mount_Alexmos::parse_body(), the CMD_GET_ANGLES_EXT case statement).